new-player/build.gradle.kts

```diff
@@ -71,12 +71,15 @@ dependencies {
     implementation(libs.androidx.ui.tooling.preview)
     implementation(libs.androidx.hilt.navigation.compose)
     implementation(libs.androidx.media3.common)
+    implementation("androidx.media3:media3-datasource-cronet:1.4.1")
     implementation(libs.coil.compose)
     implementation(libs.reorderable)
     implementation(libs.androidx.media3.session)
     implementation(libs.androidx.media3.exoplayer.dash)
     implementation(libs.androidx.adaptive.android)
 
+    implementation("org.chromium.net:cronet-embedded:119.6045.31")
```

Comment:
Because said by Audric and Stypox but now I need to remove them and just keep the important context changes as they will override the get data source obviously